wait_for_it/test_wait_for_it.py

```diff
@@ -100,14 +100,36 @@ def test_service_available(self, _label, extra_argv):
         finally:
             server.stop()
 
+    @parameterized.expand([("parallel", ["-p"]), ("serial", [])])
+    def test_service_available_impliedhost(self, _label, extra_argv):
```

Comment:
(PS: I think `impliedhost` should maybe be `implied_host` with an `_` to separate the two words.)

Reply:
Updated